public/index.html

```diff
@@ -2,8 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+   <!-- <link href="https://fonts.googleapis.com/css2?family=Bitter&display=swap" rel="stylesheet">-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<!--<link href="https://fonts.googleapis.com/css2?family=Bitter&family=Montserrat&display=swap" rel="stylesheet">-->
```

Comment:
This should actually work fine to load the fonts. No need for the webfontloader imo. Or is there any reason to use it from your side?